samples/Fc/Domain/Infrastructure.fs

```diff
@@ -0,0 +1,16 @@
+namespace global
+
+open FSharp.UMX // see https://github.com/fsprojects/FSharp.UMX - % operator and ability to apply units of measure to Guids+strings
+
+type LocationId = string<locationId>
+and [<Measure>] locationId
+module LocationId =
+    let parse (value : string) : LocationId = %value
+    let toString (value : LocationId) : string = %value
+
+type LocationEpochId = int<locationEpochId>
+and [<Measure>] locationEpochId
+module LocationEpochId =
```

Comment:
Any reason these parse/next/tostrings are  not just member functions of the type?

Reply:
The 'type', being a type alias, only exists at compile time, so there's no type to augment as such